modules/installation-creating-aws-subnet-localzone.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-creating-aws-subnet-localzone_{context}"]
+= Creating a subnet in AWS Local Zones
+
+You must create a subnet in AWS Local Zones before you configure a worker machineset for your {product-title} cluster.
+
+You must repeat the following process for each Local Zone you want to deploy worker nodes to.
+
+You can use the provided CloudFormation template and a custom parameter file to create a stack of AWS resources that represent the subnet.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in to the Local Zone group.
+
+.Procedure
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "VpcId", <3>
+    "ParameterValue": "vpc-<random_string>" <4>
+  },
+  {
+    "ParameterKey": "PublicRouteTableId", <5>
+    "ParameterValue": "<vpc_rtb_pub>" <6>
+  },
+  {
+    "ParameterKey": "LocalZoneName", <7>
+    "ParameterValue": "<cluster_region_name>-<location_identifier>-<zone_identifier>" <8>
+  },
+  {
+    "ParameterKey": "LocalZoneNameShort", <9>
+    "ParameterValue": "<lz_zone_shortname>" <10>
+  },
+  {
+    "ParameterKey": "PublicSubnetCidr", <11>
+    "ParameterValue": "10.0.128.0/20" <12>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the `install-config.yaml` file for the cluster.
+<3> The VPC created resources will belong to.
```

Comment:
```suggestion
<3> The VPC ID that the subnet will be created.
```

Reply:
@mtulio, do you mean something like "The VPC ID in which the cluster's subnet will be created?"

Reply:
@kalexand-rh the subnet is part(belongs to) of VPC, so I think something like this could be better: "The VPC ID in which the Local Zone's subnet will be created".